### DISCUSSION TITLE
CTS/RTS

### DISCUSSION THREAD

**Discussion by nelgin** [bozimmerman/Zimodem#93] [Q&A]
In the docs I see mention of RTS/CTS flow control, but I cannot seem to figure out which pins provide these signals from the hardware device. Can anyone shed some light? Thanks.

**Comment by Biker1bob** [comment]
Sure, what device are you using? ESP what? 8266? or 32?  are you using a nodemcu breakout board? or just the module?

I use ESP8266 nodemcu boards and the pins there are GPIO4 (RTS) and GPIO5(CTS)

James

**Comment by nelgin** [comment]
I'm using an  ESP8266 board from Amazon. 

 https://www.amazon.com/dp/B07SS8ZPNJ

Thanks for the info. I'll give it a try.

**Reply by Biker1bob** [reply]
remember that RTS/CTS cross over.. so from the board to the DB plug your using cross CTS/RTS

That is literally the board that I made mine out of.
![ledsmodem](https://user-images.githubusercontent.com/33105915/211214267-38ab7f0a-7b03-404a-8b8d-66cca4415628.jpg)

James
